packages/pluggableWidgets/badge-web/src/Badge.editorPreview.tsx

```diff
@@ -4,23 +4,24 @@ import { createElement, ReactElement } from "react";
 import { BadgePreviewProps } from "../typings/BadgeProps";
 import { Badge } from "./components/Badge";
 
-declare function require(name: string): string;
-
 export const preview = (props: BadgePreviewProps): ReactElement => {
-    const { class: classname, style, type, value, brandStyle, onClick } = props;
+    // TODO: Change PIW preview props typing generation to remove the ts-ignore below
+    // @ts-ignore
+    const { className, style, type, value, onClick } = props;
```

Comment:
Why this is necessary? className doesnt exist here :P

Reply:
Not according to the current typings, but `className` is what the page editor gives us. There is a story (WT-3125) to change this in PIW tools.

Reply:
I also noted in that story to fix the preview of badge-web.